src/main/java/bussiness_layer/services/IProfessorService.java

```diff
@@ -10,4 +10,5 @@
 public interface IProfessorService {
     List<EnrollmentDTO> getEnrollmentsByCourseAndGroup(String profUsername, String courseCode, Short groupCode);
     TeachingDTO getTeachingByProfessorAndCourse(String profUsername, String courseCode);
+    void updateEnrollments(List<EnrollmentDTO> enrollments);
```

Comment:
Maybe we should also consider a separate service for handling enrollments?

Reply:
I thought we're having one service per user role